### DISCUSSION TITLE
Gwennen search strings

### DISCUSSION THREAD

**Discussion by worldendDominator** [Lailloken/Exile-UI#155] [Q&A] [answered]
I can't find a way to make search strings work with Gwennen. I've calibrated her screen check, it tests positive when the gamble screen is open, string list appears on middle click in vendor and stash windows, but in hers it just says "no regex string saved".
As far as I can tell, there's no separate use case for her, so I just enabled both stash and vendor for the string I want to use. Is there a bug, or did I just miss something important?

**Comment by Lailloken** [comment, marked as answer]
Someone else brought it up recently. The gwennen feature was implemented first, and later I implemented the general search-strings, so they are two separate features. It never occurred to me to merge the two and make Gwennen a use-case in the general strings. I'll do that for the next release.
To save a string for Gwennen, you have to long-press the omni-key while her window is up. A small UI will open into which to paste the string.
EDIT: I just noticed the wiki search doesn't show anything if you search for gwennen, so I now know where the confusion is coming from.
